### DISCUSSION TITLE
How do request a list of available collections?

### DISCUSSION THREAD

**Discussion by tylere** [stac-utils/titiler-pgstac#162] [Q&A] [answered]
All of the collection-related endpoints require passing a collection_id value.
For example: `/collections/{collection_id}/tilejson.json`

How do you request a list of collections via titiler-pgstac? I would expect there would be something like:
`/collections/list`

**Comment by vincentsarago** [comment]
@tylere I didn't added a `/list` endpoint because usually users will have access to a stac-fastapi application connected to the same pgstac database. 

That's said, in eoapi, I've added a `/collections` endpoint https://github.com/developmentseed/eoAPI/blob/65ec346c31fc07ccec988baaea61629e5fcd7f10/runtime/eoapi/raster/eoapi/raster/app.py#L100-L110

**Comment by vincentsarago** [comment, marked as answer]
@tylere in the latest release `1.3.0` I've added 2 debugs endpoints `/collections` and `/collections/{collection_id}`

https://github.com/stac-utils/titiler-pgstac/blob/bb314eeb3e36d186f41c50615ee11a6c511cd12c/titiler/pgstac/main.py#L113-L133

which will be added when `TITILER_PGSTAC_API_DEBUG=TRUE`
